lib/apiservers/engine/backends/backends.go

```diff
@@ -57,6 +57,7 @@ const (
 	// RetryTimeSeconds defines how many seconds to wait between retries
 	RetryTimeSeconds        = 2
 	defaultSessionKeepAlive = 20 * time.Second
+	APITimeout              = 5 * time.Second
```

Comment:
Why 5 seconds? You need to add at least a comment to justify it for other people.
5 seconds seems very small and not particularly clear.

Reply:
Completely arbitrary.  Not sure what a good timeout should be.  I'm open to suggestions.

Reply:
Please add a comment then that it is your best guess and it may need to be tuned in the future. Also it would be good to add why use have chosen 5. Why not 15 or 30?